app/Policies/PostPolicy.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Policies;
+
+use App\Facades\UserCache;
+use App\Models\Post;
+use App\User;
+use Illuminate\Auth\Access\HandlesAuthorization;
+
+class PostPolicy
+{
+    use HandlesAuthorization;
+
+    public function template(?User $user, Post $post): bool
+    {
+        if ($post->exists === false) {
```

Comment:
I'm confused by this line, why is it returning true on a post that doesn't exist? Shouldn't that be impossible